kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -632,6 +774,20 @@ define([
                 );
             }
             this.tour.start();
+        },
+
+        uuidv4: function (a, b) {
```

Comment:
We have a UUID generator (actually, a couple, I think. Gotta clean those out sometime...) in the stack. Can add it with:

```Javascript
define(
    ['uuid'], 
    function(UUID) {
        const uuid = new UUID(4).format();
    }
);
```

Reply:
nice i didn't realize - will update in next commit